src/constants/flowConfig.ts

```diff
@@ -60,6 +67,38 @@ export const indexIDA: IndexIDAType = [
     inputIndex: 1,
     outputIndex: 0,
   },
+  {
+    input: DAIxAddress,
+    output: WETHxAddress,
+    subsidy: RICAddress,
+    subsidyIndex: 3,
+    inputIndex: 0,
+    outputIndex: 1,
+  },
+  {
+    input: WETHxAddress,
+    output: DAIxAddress,
+    subsidy: RICAddress,
+    subsidyIndex: 2,
+    inputIndex: 1,
+    outputIndex: 0,
+  },
+  {
+    input: RICAddress,
+    output: USDCxAddress,
+    subsidy: WETHxAddress,
+    subsidyIndex: 3,
+    inputIndex: 1,
+    outputIndex: 0,
+  },
+  {
+    input: USDCxAddress,
+    output: RICAddress,
+    subsidy: WETHxAddress,
+    subsidyIndex: 2,
+    inputIndex: 0,
+    outputIndex: 1,
+  },
```

Comment:
I think these might be off, I tried to open a stream USDC>>RIC and got this error:
<img width="1325" alt="Screen Shot 2022-03-04 at 7 31 57 PM" src="https://user-images.githubusercontent.com/5077880/156859472-ebf90ea0-3fec-40c5-9041-6d628112d8d9.png">

Reply:
https://dashboard.tenderly.co/tx/polygon/0x1fc6c3f5b048ecf52159e45a570d5661fa366c8ea0064f2391a443e09f59032a